src/SwiftUI/SwiftUI/Views/ViewModifier.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	[SwiftImport (SwiftUILib.Path)]
+	public abstract unsafe record ViewModifier<T> : View
+	{
+		public abstract T Body (Content content);
```

Comment:
Just like with `View`, if `ViewModifier` is not generic, we won't be able to have an abstract member for `Body` .. we'll have to rely on reflection to get it